### ISSUE TITLE
cf registered-endpoints doesn't show the endpoints registered with internal port

### ISSUE THREAD

**Issue by erboucherie** [pivotal-cf/metric-registrar-cli#38]
Hello , we are actively using the new feature (internal ports) because this is the one which guarantees that you can have metrics without exposing publicly an endpoint .
Unfortunately , the _cf registered-edpoints_ commands doesn't show any endpoints which has been registere with the **--internal-port** argument .
This is annoying because the only way to see if such endpoint has been set is with the cf services command which shows the service instance but not the path :  
`m123182@AFR712998:/c/Users/m123182$ cf register-metrics-endpoint mercury-ws /actuator/prometheus --internal-port 8082
m123182@AFR712998:/c/Users/m123182$ cf registered-metrics-endpoints
App  Path 
m123182@AFR712998:/c/Users/m123182$ cf services
Getting services in org mercury / space DEV as m123182...

name                                       service             plan       bound apps             last operation     broker              upgrade available

secure-endpoint-8082-actuator-prometheus   user-provided                  mercury-ws`

**Comment by MiliDurasovic** [comment]
I just ran into the exact same issue 
```
cf register-metrics-endpoint python-sample-custom-metrics /metrics --internal-port 8080
```
```
echo $?
0
```
```
cf registered-metrics-endpoints
App  Path
```

**Comment by MiliDurasovic** [comment]
@ctlong 
The bug seems to be purely visual.
After registering the metrics-endpoint with an internal-port, even though I cannot list it with `cf registered-metrics-endpoints`, I was still able to access the metrics via AppMetrics over the registered endpoint.

**Comment by ctlong** [comment]
@MiliDurasovic you are correct that the bug appears purely cosmetic. It seems the `registered-metrics-endpoints` was never updated to fetch and list secure metric endpoint registrations.

**Comment by MiliDurasovic** [comment]
Thanks a lot for the quick fix.
I can confirm that it looks good now with release 1.4.0:

```
cf register-metrics-endpoint python-sample-custom-metrics /metrics --internal-port 8080
```

```
cf registered-metrics-endpoints
App                           Path
python-sample-custom-metrics  :8080/metrics
```
